components/script/dom/htmlimageelement.rs

```diff
@@ -432,16 +433,17 @@ impl HTMLImageElement {
             _ => {},
         };
 
-        let map = self.upcast::<Node>()
-                      .following_siblings()
+        let useMapElements =  self.upcast::<Node>()
+                      .GetRootNode()
```

Comment:
Use the document here as I mentioned in my previous comment, also the indentation is wrong.